03_launch_mgmt_cluster.sh

```diff
@@ -319,7 +266,7 @@ function update_capm3_imports(){
 }
 
 #
-# Update the images for the CAPM3 deployment file to use local ones
+# Update the images for CAPM3 and BMO deployment file to use local ones
```

Comment:
# Update CAPM3 and BMO manifest to use the the local registry for images

Reply:
It is not just for registry. For example, we use this to get an old release of CAPM3 for upgrade tests.

Reply:
Updated